scss/application.scss

```diff
@@ -5,8 +5,8 @@
 @import 'book-list-detail';
 @import 'login-detail';
 
-body {
-    font-family: 'Roboto', sans-serif;
+.body {
```

Comment:
Does the project have an element with `class="body"` ? Because if not this class wont be applied